### PR TITLE
Fixes broken unit tests

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Unit test file `t/test01.t` leaks queries when network is set to off. A change in .fr NS and DS triggered the tests to break. This PR:

* Updates `t/test01.t` with correct expectations on .fr.
* Updates `t/test01.t` so that it does not leak queries.
* Updates `t/test01.data` to match updates on .fr and updated `t` file.
* Updates `t/idn.t` so that it does not leak queries.

## Context

Issue #1191

## How to test this PR

1. Unit tests must pass.
2. Review.
3. Verify that no queries are sent when network is set to off.

### CODE REVIEW

File: t/test01.t [hunk 1]
```diff
@@ -190,16 +203,14 @@ subtest 'API calls' => sub {
...
         ok( grep( /2001:678:4c::1/, @ip_list ), 'Has "2001:678:4c::1" ip' );
-        ok( grep( /193\.176\.144\.22/, @ip_list ), 'Has "193.176.144.22" ip' ); # e.ext.nic.fr
-        ok( grep( /2a00:d78:0:102:193:176:144:22/, @ip_list ), 'Has "2a00:d78:0:102:193:176:144:22" ip' );
         ok( grep( /194\.146\.106\.46/, @ip_list ), 'Has "194.146.106.46" ip' ); # f.ext.nic.fr
         ok( grep( /2001:67c:1010:11::53/, @ip_list ), 'Has "2001:67c:1010:11::53" ip' );
 
         my $ds_value = {
             'algorithm' => 13,
-            'digest' => '1b3386864d30ccc8f4541b985bf2ca320e4f52c57c53353f6d29c9ad58a5671f',
+			'digest' => '1303e8da8fb60db500d5bea1ee5dc9a2bcc93dfe2fc43d346576658feccf5749', # must match case
```
Comment (marc-vanderwal) [inline t/test01.t:211]: Best to keep the indentation as it was, I think (the rest is indented with spaces, here it’s being replaced with tabs):
```suggestion
            'digest' => '1303e8da8fb60db500d5bea1ee5dc9a2bcc93dfe2fc43d346576658feccf5749', # must match case
```

**Review by marc-vanderwal — COMMENTED** [review]
Looks fine, except for one teeny tiny little thing.

File: t/test01.t (hunk 1 above)
Comment (matsduf) [reply t/test01.t:211]: My emacs wants to have it more indented.

**Review by marc-vanderwal — APPROVED** [review]

**Comment by matsduf** [comment]
> After investigating, I would suggest an alternative fix:

I suggest that you create a new PR instead.

**Comment by mattias-p** [comment]
It's a bit awkward, but it looks right to me. Not saying it's your fault that it's awkward.

**Pull request merged**